content/docs/command-reference/fetch.md

```diff
@@ -120,8 +120,8 @@ or `-T` options are used).
 
 ## Examples
 
-Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
-pipeline stages, such as the <abbr>DVC project</abbr> created for the
+Let's employ a <abbr>workspace</abbr> with some data, code, ML models, pipeline
+stages, such as the <abbr>DVC project</abbr> created for the
```

Comment:
Same here: new workspace; fresh workspace; clean workspace — something like that